cli/src/commands/token/create/mod.rs

```diff
@@ -2,17 +2,21 @@ use anyhow::Result;
 use clap::Parser;
 
 pub mod dev;
+pub mod pub_ns;
 
 #[derive(Parser)]
 pub enum SubCommand {
 	#[clap(alias = "dev")]
 	Development(dev::Opts),
+	#[clap(alias = "public", alias = "pub")]
+	PublicNamespace(pub_ns::Opts),
```

Comment:
Are these meant for internal use with engine plugins? Could use some docs indicating what they do

Reply:
See v3